scalardb-graphql-sample/README.md

```diff
@@ -1,130 +1,222 @@
+Now that you have created two accounts, you can send funds from one account to the other account.
 
-- `graphql/` - The GraphQL documents (queries and mutations) for the app functions are located here. The GraphQL Code Generator (`graphql-codegen`) reads them to generate the client TypeScript code. Read the next section about the generator.
-- `src/` - All TypeScript code are located here.
-  - `src/generated/graphql.ts` - The generated file with `graphql-codegen` based on the GraphQL endpoint and the local documents in the `graphql/` directory.
-  - `emoney.ts` - The main file of the client app
-- `codegen.yml` - The configuration file for `graphql-codegen`
+To have `user1` pay `5` to `user2`, run the following command:
 
-### GraphQL Code Generator
+```console
+$ node dist/emoney.js pay user1 user2 5
+```
+
+### Get an account balance
+
+After sending funds from one account to the other, you can check the balance of each account.
+
+To get the balance of `user1`, run the following command:
+
+```console
+$ node dist/emoney.js show user1
+```
+
+To get the balance of `user2`, run the following command:
+
+```console
+$ node dist/emoney.js show user2
+```
 
-The client code for this app was generated with [GraphQL Code Generator](https://www.graphql-code-generator.com/).
+### Stop the sample application
 
-It is configured with the `codegen.yml` file. This file enables three plugins to generate `src/generated/graphql.ts`.
+To stop the sample application, stop the Docker container by running the following command:
 
-- `typescript` - An essential plugin to generate the TypeScript types based on the GraphQL schema.
-- `typescript-operations` - Generates TypeScript types based on the schema and the documents in the `graphql` directory.
-- `typescript-graphql-request `- Generates the client code that is called "SDK". With the functions provided by the SDK, we can call GraphQL operations defined in the document easily.
+```console
+$ docker-compose down
+```
 
-Note that the `rawRequest: true` configuration should be set for the `typescript-graphql-request ` plugin as in `codegen.yml`. This is because we need to access the `extensions` key in the GraphQL response where the the transaction ID is returned from the server.
+## Reference - Application structure
 
-To generate the code again, you can run:
+This section describes how this client application is configured in detail.
 
-````console
-npm run generate
-````
+### Files and directories
 
-## Clean up
+The following list is a description of the key files and directories in this application:
 
-To stop the services and remove the volume for the Cassandra data, run the following commands:
+- `graphql`. The GraphQL documents (queries and mutations) for the application functions are located in this directory. The GraphQL Code Generator (`graphql-codegen`) reads the documents to generate the client TypeScript code. For more details, see [GraphQL Code Generator](#graphql-code-generator).
```

Comment:
```suggestion
- `graphql`. The GraphQL schemas (queries and mutations) for the application functions are located in this directory. The GraphQL Code Generator (`graphql-codegen`) reads the schemas to generate the client TypeScript code. For more details, see [GraphQL Code Generator](#graphql-code-generator).
```
According to https://graphql.org/learn/schema/, "schemas" sounds better to me instead of "documents". But I'm not familiar with GraphQL, so any feedback is welcome

Reply:
Ah, that's good to know! I kept the wording `document` as is since I found the following reference: [GraphQL Document](https://spec.graphql.org/June2018/#:~:text=A%20GraphQL%20Document%20describes%20a,of%20a%20GraphQL%20type%20system). However, that reference seems a bit outdated (2018).

If `schema` is more commonly used these days, I'm open to changing the wording👍